games-engines/frotz/frotz-2.50.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A Portable Z-Machine Interpreter"
+DESCRIPTION="Frotz is an interpreter for Infocom games \
+and other Z-machine games. \
+It complies with standard 1.0 of Graham Nelson's specification. \
+It was written by Stefan Jokisch in 1995-1997"
+HOMEPAGE="https://davidgriffith.gitlab.io/frotz/"
+COPYRIGHT="1995-1997 Stefan Jokish
+		1997-2019 David Griffith"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/DavidGriffith/frotz/-/archive/$portVersion/frotz-$portVersion.tar.bz2"
+CHECKSUM_SHA256="7fd1185318dec2c897228d6ed81d4d2599d64d1e5bb7f913c09d17f3337ec663"
+PATCHES="frotz-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	frotz$secondaryArchSuffix = $portVersion
+	cmd:frotz
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libao$secondaryArchSuffix
+	lib:libmodplug$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libsamplerate$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libao$secondaryArchSuffix
+	devel:libmodplug$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libsamplerate$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
```

Comment:
Maybe add "cmd:git" here also

Reply:
Okay, I see now.  I thought Haiku was able to use `libao` for audio output.  That's why I put the check where I did.  It checks to see if `libao` should be used.  Then it checks if the OS is Haiku.  If it isn't Haiku, then `CLFLAGS` is left alone.  Looking at https://github.com/xiph/libao/pull/5, it seems like support for Haiku in `libao` has been languishing since early 2017.

Reply:
This one is for @korli , korli you created the issue there, is this still valid?

Reply:
@DavidGriffith not sure I understand what you mean by "has been languishing", it's an upstream problem. a Haiku patched libao is present in the HaikuPorts repository.

Reply:
I just looked at the official `libao` repository and noticed that Haiku support has yet to be merged in.